assets/js/show-audio.js

```diff
@@ -57,10 +57,10 @@
         requestAnimationFrame(renderFrame);
         // update data in frequencyData
         analyser.getByteTimeDomainData(frequencyData);
-        var high = frequencyData.reduce((a, v, i) => v > a ? v : a, 128);
-        var low = frequencyData.reduce((a, v, i) => v < a ? v : a, 128);
-        var offsetHigh = high / 128;
-        var offsetLow = (128 - low + 128) / 128;
+        const high = frequencyData.reduce((a, v, i) => v > a ? v : a, 128);
+        const low = frequencyData.reduce((a, v, i) => v < a ? v : a, 128);
+        const offsetHigh = high / 128;
+        const offsetLow = (128 - low + 128) / 128;
```

Comment:
`128 + 128` ? I'm guessing this is to "show your work"? I'm not sure. if I understand `getByteTimeDomainData`, that's a byte range from -1 to 1, so 128 is `0`?